browser/iframe-browser.js

```diff
@@ -60,23 +70,14 @@ define(function (require, exports, module) {
      * In which our iFrame will exists and will be filled
      * with the url that has been passed to this function
      */
-    function _update(url) {
-        // Empty all contents of #second-pane
-        var _panel = $("#second-pane").empty();
-
-        // Create the iFrame for the blob to live in
-        var iframeConfig = {
-            id: "bramble-iframe-browser",
-            frameborder: 0
-        };
-
+    function update(url) {
         // If url has been sent, make the iFrames' src that of the url
         if(url) {
-            iframeConfig.src = url;
+            var $iframe = $("#bramble-iframe-browser");
+            if ( $iframe.length ) {
+                $iframe.attr('src',url);
```

Comment:
Watch your spaces around arguments.  You want to have the parens be tight to the args, but have a space after commas.